src/cloud-cli/userdb.ts

```diff
@@ -199,14 +202,16 @@ export function rollbackmigration() {
   const rollbackcommands = configFile.database.rollback;
   
   try {
-    rollbackcommands?.forEach((cmd) => {
+        rollbackcommands?.forEach((cmd) => {
```

Comment:
Format is a bit off: we use 2 spaces everywhere

Reply:
I applied `prettier` to the entire file in [4b978d7](https://github.com/dbos-inc/dbos-ts/pull/207/commits/4b978d795d300c31d1a8aefcda6c7f64b69c14fb)